resources/geocoding/he/972.txt

```diff
@@ -0,0 +1,22 @@
+# Copyright (C) 2017 The Libphonenumber Authors
+
+# Licensed under the Apache License, Version 2.0 (the "License");
+# you may not use this file except in compliance with the License.
+# You may obtain a copy of the License at
+
+# http://www.apache.org/licenses/LICENSE-2.0
+
+# Unless required by applicable law or agreed to in writing, software
+# distributed under the License is distributed on an "AS IS" BASIS,
+# WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+# See the License for the specific language governing permissions and
+# limitations under the License.
+
+# Sourced from https://he.wikipedia.org/wiki/קידומת_טלפון_בישראל
+
+9722|ירושלים
+9723|תל אביב-יפו והמרכז
```

Comment:
"והמרכז" = "and central" which the english wiki says too -- https://en.wikipedia.org/wiki/Telephone_numbers_in_Israel#Geographic_area_codes -- but central/northern/southern isn't in the english version.

From the english wiki it sounds like [tel aviv](https://en.wikipedia.org/wiki/Tel_Aviv_District) is the district, but "[tel aviv-yafo](https://en.wikipedia.org/wiki/Tel_Aviv)" is the city.
So, it seems to match the pattern of "City + surrounding".